### ISSUE TITLE
Using install-app-deps on ARM

### ISSUE THREAD

**Issue by r4zzz4k** [electron-userland/electron-builder#2092]
* **Version**: 19.28.4
* **Target**: arm

I'm trying to build an app with some native dependencies on Raspberry Pi 3. While running `install-app-deps` it correctly detects target arch (`default: "arm"`), but it's not listed in available architectures. I know that `electron-builder build` supports `--armv7l` flag, but is it possible to retarget native dependencies to Electron's Node?

```
node_modules/.bin/electron-builder install-app-deps

Options:
  --help      Show help                                                [boolean]
  --platform  The target platform
                        [choices: "linux", "darwin", "win32"] [default: "linux"]
  --arch      The target arch   [choices: "ia32", "x64", "all"] [default: "arm"]

Invalid values:
  Argument: arch, Given: "arm", Choices: "ia32", "x64", "all"
```

**Comment by r4zzz4k** [comment]
Tried to add `arm` manually to `install-app-deps.ts`, it seemed to work, but I'm not sure if it actually rebuilt libraries. `node-java`, which I'm using, seem to support [only `armv6`](https://github.com/joeferner/node-java/blob/master/README.md#installation-arm-raspberry-pi), but modified `electron-builder install-app-deps` finished successfully (and `electron-builder build` failed as expected).
By the way, is there a way to skip specific depdendency rebuilding? I'm not seeing anything useful in config docs.

**Comment by develar** [comment]
Will be fixed 22 September.

**Comment by r4zzz4k** [comment]
Cool, thank you!

**Comment by elboletaire** [comment]
Any news on this?

**Comment by r4zzz4k** [comment]
Sorry for the long delay. I've written simple JS script which basically passing the following flags to `electron-builder` based on current arch:
```js
const isArm = os.arch().startsWith('arm')
// Flag for install-app-deps:
isArm ? ' --arch all' : ' --arch x64'
// Flag for build:
isArm ? ' --armv7l' : ' --x64'
```
Somehow I managed to build the app for Raspberry, but building process for my project is messy overall, and I didn't have time to investigate much further (you know, tight deadlines as usual). Due to dependency on `node-java`, which is itself a bit strange in regards of finding needed libraries, I had to have custom build script anyway.
